repobuilder/job.go

```diff
@@ -207,7 +207,11 @@ func (j *repoBuilderJob) setup() {
 		}
 	}
 
-	j.tmpdir, err = ioutil.TempDir("", j.ID())
+	if err = os.MkdirAll("/data/tmp"); err != nil {
+		j.AddError(errors.Wrap(err, "could not create temp directory"))
+		return
+	}
```

Comment:
what if this does exist?

Reply:
If it exists already, doesn't `MkdirAll` just no-op?

Reply:
I think it's right but you have to put perms?